hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/ipc/TestRPC.java

```diff
@@ -1219,9 +1218,8 @@ public void testClientBackOff() throws Exception {
     server = setupTestServer(builder);
 
     @SuppressWarnings("unchecked")
```

Comment:
We can remove `@SuppressWarnings("unchecked")` here

Reply:
I will fix it.